### ISSUE TITLE
finetune accuracy

### ISSUE THREAD

**Issue by kli-nlpr** [deepinsight/insightface#350]
I use the pretrained model https://github.com/deepinsight/insightface/wiki/Model-Zoo#31-lresnet100e-irarcfacems1m-refine-v2 to finetune on the same ms1m-refine-v2 dataset(https://github.com/deepinsight/insightface/wiki/Dataset-Zoo#ms1m-refine-v2recommended)
The accuracy start from 0, is this normal?
The command line is as follows
```
CUDA_VISIBLE_DEVICES='0,1,2,3' python -u train_softmax.py --network r100 --loss-type 4 --margin-m 0.5 --data-dir ../datasets/faces_emore  --prefix ../model-r100 --per-batch-size 64  --pretrained ../pre_trained_models/model-r100-ii/model,0
```
Thanks

**Comment by nttstar** [comment]
It's normal, as I removed the fc7 layer weight in all pretrained model.

**Comment by nttstar** [comment]
Try decreasing learning rate and also it is not recommended to fine-tune the same dataset.

**Comment by LCorleone** [comment]
@mingzhang96 +1 
Have you found the solutions?
